src/styles/blocks/stars.scss

```diff
@@ -0,0 +1,36 @@
+.stars {
+  display: flex;
+  justify-content: space-between;
+  padding-bottom: 24px;
+
+  &__rating-box {
+    display: flex;
+  }
+
+  &__star {
+    height: 16px;
+    width: 16px;
+    background-image: url("../images/star.svg");
+    background-repeat: no-repeat;
+    background-position: center;
+  }
+
+  &__star:not(:last-child) {
+    margin-right: 4px;
+  }
+
+  &__reviews {
+    margin: 0;
+    font-size: 10px;
+    line-height: 14px;
+    color: $mainTextColor;
+    font-style: $fontStyle;
+    display: flex;
+    align-items: flex-end;
+    text-align: right;
+  }
+
+  &__star:nth-child(-n + 4) {
```

Comment:
Consider keeping functionality for modifiers from the previous task

Reply:
Request is little bit unclear